pkgs/by-name/da/davis/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib,
+  fetchFromGitHub,
+  php,
+}:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "davis";
+  version = "4.4.1";
+
+  src = fetchFromGitHub {
+    owner = "tchapi";
+    repo = "davis";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-UBekmxKs4dveHh866Ix8UzY2NL6ygb8CKor+V3Cblns=";
+  };
+
+  patches = [
+    # Symfony loads .env files from the same directory as composer.json
+    # The .env files contain runtime configuration that shouldn't be baked into deriviation for the package
+    # This patch adds a few extension points exposing three environment variables:
+    #    RUNTIME_DIRECTORY (where to load .env from), CACHE_DIRECTORY and LOG_DIRECTORY (symfony cache and log rw directories)
+    ./davis-data.patch
```

Comment:
What is the source of this patch? If the source is in AUR or some other site like that we usually use fetchpatch, goal is to avoid adding patche files  to nixpkgs unless necessary. Then, it's important to provide the reference of source material (as in a comment with url).

Reply:
I wrote the patch, so the source is me.

Reply:
.. added a link to the upstream PR